### PR TITLE
Interactivity fixes, p.I

### PR DESCRIPTION
**Pull request by shamansir**

### CODE REVIEW

File: sandbox/interactivity-sandbox.js [hunk 1]
```diff
@@ -161,10 +166,20 @@ function sandbox() {
...
 }
 
 function load_last_code() {
     if (!localStorage) throw new Error('Local storage support required');
-    return localStorage.getItem('_current_code') || '';
+    return localStorage.getItem('_i_current_code') || '';
+}
+
+function save_current_json(json) {
+    if (!localStorage) throw new Error('Local storage support required');
```
Comment (mcm69) [inline sandbox/interactivity-sandbox.js]: http://caniuse.com/#search=localstorage
don't think that is necessary. also, in the event `localStorage` is not defined, you'll get a `ReferenceError` for an undefined element here, so `window.localStorage` will have to be used.

Comment (shamansir) [inline sandbox/interactivity-sandbox.js]: true. removed it. thanks!

File: src/anm/animation/animation.js [hunk 2]
```diff
@@ -356,15 +357,43 @@ Animation.prototype.unsubscribeEvents = function(canvas) {
 };
 
 Animation.prototype.handle__x = function(type, evt) {
+    var anim = this;
     if (events.mouse(type)) {
         var pos = evt.pos;
-        this.each(function(child) {
+        var foundTarget = false;
+        anim.each(function(child) {
```
Comment (shamansir) [inline src/anm/animation/animation.js]: This algorithm confuses me with it's even though it checks bounds first and mouse events are passed only if `options.handleEvents` is `true` and if element is subscribed to them. @mcm69 Probably you have some thoughts on improvement?

File: sandbox/interactivity-sandbox.js (hunk 1 above)
Comment (RobertoUa) [inline sandbox/interactivity-sandbox.js]: also some people can disable localStorage in browser settings. Also there's could be problems in some browsers in incognito mode.

Comment (shamansir) [inline sandbox/interactivity-sandbox.js]: It's just a page for internal testing, it's not a big deal! :)

Comment (mcm69) [inline sandbox/interactivity-sandbox.js]: i see someone is bored in Asia ;)

Comment (skavish) [inline sandbox/interactivity-sandbox.js]: @RobertoUa go back to the beach :)

**Pull request merged**